### PR TITLE
feature: pass title at har page creation

### PR DESCRIPTION
**Pull request by nicolasunravel**
this time with atomic commits, and proper messages

### CODE REVIEW

File: browsermobproxy/client.py [hunk 1]
```diff
@@ -97,22 +99,24 @@ def har(self):
         return r.json()
 
 
-    def new_har(self, ref=None, options={}):
+    def new_har(self, ref=None, title=None, options=None):
```
Comment (AutomatedTester) [inline browsermobproxy/client.py]: Please put `title` at the end of the argument list. If people arent using this as kwargs it is going to break things.

**Comment by nicolasunravel** [comment]
don't know why tests failed, bmp init issue on the CI ?

**Comment by AutomatedTester** [comment]
I have gone through and made some fixes for some tests (because I had been
lazy). If you rebase and they still fail, let me know and I will have a
gander.

Thanks again for the patch!

David

David Burns
Email: david.burns@theautomatedtester.co.uk
URL: http://www.theautomatedtester.co.uk/

**Comment by alan-unravel** [comment]
Hi, @nicolasunravel  is OOF for the next while and he asked me to throw an eye on this. He rebased as far as I am aware and the CI seems to fail but I am unsure of how to resolve this.

**Comment by AutomatedTester** [comment]
@alan-unravel Looking at the errors, the rebase didnt work. I will rebase/cherry-pick for you now. I also have the tests passing locally.

**Comment by AutomatedTester** [comment]
landed in cfda250, b748e8e, 9840668, 728dd46

**Comment by AutomatedTester** [comment]
thanks for the patch!

**Pull request closed without merging**